docs/components/mappers/modbus_mapper_v2.md

```diff
@@ -0,0 +1,76 @@
+# Modbus Mapper V2
+
+# Introduction
+The Modbus mapper is an interface between KubeEdge and Modbus devices. It could set/get device data, get and report the device status.
+In this version, Modbus TCP and RTU mode are supported.
+ 
+## Running the mapper
+
+  1. Please ensure that the Modbus device is connected to your edge node
+  2. Set 'modbus=true' label for the node (This label is a prerequisite for the scheduler to schedule modbus_mapper pod on the node)
+
+      ```shell
+      kubectl label nodes <name-of-node> modbus=true
+      ```
+
+  3. Build and deploy the mapper by following the steps given below.
+
+### Building the Modbus mapper
+
+ ```shell
+cd $GOPATH/src/github.com/kubeedge/kubeedge/
+make modbusmapper or `make mappers` if you want to build all mappers
+docker tag modbusmapper:v1.0 <your_dockerhub_username>/modbusmapper:v1.0
```

Comment:
should it be modbusmapper:v2.0 or modbusgomapper:v1.0?

Reply:
Err, the version in the deployment example file is modbusmapper:v1.0.